### ISSUE TITLE
Read receipts not accessible with VoiceOver

### ISSUE THREAD

**Issue by sempruijs** [vector-im/element-x-ios#1139]
### Steps to reproduce

Turn on VoiceOver and open an conversation, so you can read the messages.
When you put the VoiceOver cursor on the read receipt, (that round checkmark) VoiceOver doesn't give a clear description of what that circle means.

#### What did you expect?

I expected the following behavior:
(not read) =  "Message delivered"
(read by @harveyharvest:matrix.org with display name "Harvey") = "Read by Harvey"
(read by multiple users with display names "Stan" "Leo" "Harvey") = "Read by Stan, Leo and Harvey."

#### What happened instead?

I found an "unlabeled" image. This is not useful for Visually impaired users.

### Your phone model

iPhone 10s

### Operating system version

iOS 16.5

### Application version

1.1.3

### Homeserver

matrix.org

### Will you send logs?

No
